### PR TITLE
Lazy-import boto3

### PR DESCRIPTION
**Pull request by normanrz**
**Proposed changes**:
- Lazy-import for `boto3` and `botocore` since they are only used for persisting the model to S3.

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for submitting this PR, we'll give it a review as soon as possible

**Comment by tmbo** [comment]
please also add the change to the `CHANGELOG.rst`

File: alt_requirements/requirements_bare.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 gevent==1.2.2
 klein==17.10.0
 hyperlink==17.3.1
-boto3==1.5.20
```
[the remark this answers is not in the document]
Comment (normanrz) [reply alt_requirements/requirements_bare.txt]: good point ;-)

**Comment by tmbo** [comment]
looks good, thanks for the changes :tada:

**Pull request merged**